easybuild/easyconfigs/y/yaff/yaff-1.4.2-intel-2017b-Python-2.7.14.eb

```diff
@@ -11,15 +11,18 @@ toolchain = {'name': 'intel', 'version': '2017b'}
 
 source_urls = ['https://github.com/molmod/yaff/releases/download/%(version)s']
 sources = [SOURCE_TAR_GZ]
-checksums = ['f61a85dcbf375acb21d7540ad7a80850996315d5eca608cfe2d57867f6120710']
+patches = ['yaff-%(version)s-add-missing-include.patch']
```

Comment:
Hmm, this worked fine without this patch though?

Reply:
I really have no idea. I also don't understand why it passed the unit tests without setting `lowopt=True`. I'll check.

Reply:
This is weird! Indeed, the patch is not needed for `yaff-1.4.2-intel-2017b-Python-2.7.14.eb` but it is still needed to let the tests pass for `yaff-1.4.2-intel-2017b-Python-3.6.3.eb`. Even stranger, the tests run fine on Delcatty without `lowopt=True`, while on Golett `lowopt=True` is needed. I'll revert the changes in `yaff-1.4.2-intel-2017b-Python-2.7.14.eb` but I can't make much sense out of the current behavior.

Reply:
@tovrstra Delcatty is Intel Sandy Bridge, Golett is Intel Haswell, which may explain why `lowopt=True` is only needed for the latter for `yaff`.